### PR TITLE
add standalone orca doc

### PR DESCRIPTION
**Pull request by jdamiba**
The purpose of this PR is to add a standalone doc about exporting charts as static images to the `plotly fundamentals` section of the R graphing libraries docs.

### CODE REVIEW

File: r/2020-01-20-static-image-export.Rmd [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+### Export R Graphs As Static Images Using `orca()`
+
+[Orca](https://github.com/plotly/orca) is an open source command line tool for generating static images of graphs created with [Plotly's graphing libraries](https://plot.ly/graphing-libraries). 
+
+To use Orca to export static images of the graphs you create with the `plotly` R package, you can use the built-in `orca()` funciton in versions `4.7.900` and above.
+
+In order to use the `orca()` function, you need to have the [`processx`](https://github.com/r-lib/processx) R package installed. 
+
+The `orca()` function accepts two parameters. The frist is the plot to be exported and second is the filename.
```
Comment (nicolaskruchten) [inline r/2020-01-20-static-image-export.Rmd]:
```suggestion
The `orca()` function accepts two parameters. The first is the plot to be exported and second is the filename. 
```

File: r/2020-01-20-static-image-export.Rmd [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+### Version Check
+
+Version 4 of Plotly's R package is now [available](https://plot.ly/r/getting-started/#installation)!<br>
+Check out [this post](http://moderndata.plot.ly/upgrading-to-plotly-4-0-and-above/) for more information on breaking changes and new features available in this version.
+```{r}
+library(plotly)
+packageVersion('plotly')
+```
+
+### Export R Graphs As Static Images Using `orca()`
```
Comment (nicolaskruchten) [inline r/2020-01-20-static-image-export.Rmd]: we'll want to add some of the extra information here about how orca runs locally, doesn't require any kind of network access, and the various file formats that you can target like jpg, svg, png, pdf, eps etc,

Comment (nicolaskruchten) [reply r/2020-01-20-static-image-export.Rmd]: also we'll need the warning from the other page about the limitations of vector vs raster

File: r/2020-01-20-static-image-export.Rmd [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+display_as: file_settings
+language: r
+layout: base
+name: Exporting Graphs as Static Images
+order: 21
+output:
+  html_document:
+    keep_md: true
+page_type: u-guide
+permalink: r/image-export-orca/
```
Comment (nicolaskruchten) [inline r/2020-01-20-static-image-export.Rmd]: This one should actually get the `/r/static-image-export/` permalink and the other page should get a new one like `/r/chart-studio-image-export/`

**Comment by jdamiba** [comment]
I have updated this PR with info about file formats and a note about network access.

File: r/2020-01-20-static-image-export.Rmd [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    keep_md: true
+page_type: u-guide
+permalink: r/static-image-export/
+thumbnail: thumbnail/sizing.png
+---
+
+```{r, echo = FALSE, message=FALSE}
+knitr::opts_chunk$set(message = FALSE, warning=FALSE)
+```
+### New to Plotly?
```
Comment (nicolaskruchten) [inline r/2020-01-20-static-image-export.Rmd]: let's remove this "new to plotly" section here so we don't forget it in the other PR

**Comment by nicolaskruchten** [comment]
💃

**Pull request merged**